internal/service/advancedclustertpf/resource_schema.go

```diff
@@ -407,6 +407,20 @@ func ResourceSchema(ctx context.Context) schema.Schema {
 				MarkdownDescription: "Storage capacity of instance data volumes expressed in gigabytes. Increase this number to add capacity.\n\n This value must be equal for all shards and node types.\n\n This value is not configurable on M0/M2/M5 clusters.\n\n MongoDB Cloud requires this parameter if you set **replicationSpecs**.\n\n If you specify a disk size below the minimum (10 GB), this parameter defaults to the minimum disk size value. \n\n Storage charge calculations depend on whether you choose the default value or a custom value.\n\n The maximum value for disk storage cannot exceed 50 times the maximum RAM for the selected cluster. If you require more storage space, consider upgrading your cluster to a higher tier.",
 			},
 			"advanced_configuration": AdvancedConfigurationSchema(ctx),
+			"pinned_fcv": schema.SingleNestedAttribute{
```

Comment:
I don't think this will be compatible with SDKv2 schema definition?

Reply:
Do you mean because it is not using a block attribute? I saw that in tfp implementation currently there are no blocks defined (everything is with attributes), preferred to maintain consistency but let me know if we prefer to leave this as a block directly. cc: @lantoli

Reply:
I saw this a bit late, but yes, I think we should try to avoid blocks for new attributes

Reply:
AFAK attributes are only supported in TPF, meaning that for supporting in existing `cluster` and `advanced_cluster` it would have to be a block